core.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+var time_interval = 500;
+var counter = 0, ok_value = 0, ng_value = 0;
+
+function changeFileName() {
+    var image1 = new Image();
+    var image2 = new Image();
+    var imageName1 = counter + "_OK.jpg";
+    var imageName2 = counter + "_NG.jpg";
+    image1.src = imageName1;
+    if (image1.width === 0) { //there is no ok image

```

Comment:
You hack! But as long as it works, I'm fine :v

Reply:
Haha :)